.circleci/config.yml

```diff
@@ -30,26 +34,59 @@ jobs:
   test-linux:
     docker:
       - image: cimg/base:stable
+    parameters:
+      node-version:
+        description: Node.js version to install
+        type: string
     steps:
-      - install
+      - install:
+          node-version: << parameters.node-version >>
       - test
   test-mac:
     macos:
       xcode: "13.3.0"
+    parameters:
+      node-version:
+        description: Node.js version to install
+        type: string
     steps:
-      - install
+      - install:
+          node-version: << parameters.node-version >>
       - test
   test-windows:
     executor:
       name: win/default
       shell: bash.exe
+    parameters:
+      node-version:
+        description: Node.js version to install
+        type: string
     steps:
-      - install
+      - install:
+          node-version: << parameters.node-version >>
       - test
 
 workflows:
   test_and_release:
     jobs:
-      - test-linux
-      - test-mac
-      - test-windows
+      - test-linux:
+          matrix:
+            parameters:
+              node-version:
+                - 18.14.0
+                - 16.19.0
+                - 14.19.0
```

Comment:
Is there a reason we'd want to do EOL versions? I'd probably not unless there's a compelling reason

Reply:
I think we want to test against versions that this package says it supports, which according to `package.json` include Node.js v14.

https://github.com/electron/chromedriver/blob/dc1c43c70217778fbcf99e049245933a99f52456/package.json#L14-L16